e2e/e2e_tests.py

```diff
@@ -39,105 +46,118 @@ def _stop_containers():
     docker_compose.clean()
 
 
-class TestE2E(unittest.TestCase):
-    COOKIES = dict()
-    CSRF_TOKEN = ''
-
-    def _login(self, next=None):
-        login_url = self._make_link('/login/')
-        client = requests.session()
-        client.get(login_url)
+class SeleniumTests(unittest.TestCase):
+    wait_time_seconds = datetime.timedelta(minutes=5).total_seconds()
+    excerpt_data = {
+        'send_keys': {
```

Comment:
Can we use a higher-level semantic key name, here, e.g. `form_text_inputs`. That the inputting is done with selenium's `send_keys` is irrelevant for the content of this sub-dict.